cmd/create/cluster/cmd.go

```diff
@@ -3330,3 +3332,34 @@ func getExpectedResourceIDForAccRole(hostedCPPolicies bool, roleARN string, role
 
 	return strings.ToLower(fmt.Sprintf("%s-%s-Role", rolePrefix, accountRoles[roleType].Name)), rolePrefix, nil
 }
+
+func getInitialValidSubnets(awsClient aws.Client, reporter *reporter.Object) ([]*ec2.Subnet, error) {
+	initialValidSubnets := []*ec2.Subnet{}
+	allSubnets, err := awsClient.GetSubnetIDs()
+	if err != nil {
+		return initialValidSubnets, err
+	}
+	for _, subnet := range allSubnets {
+		hasRHManaged := false
+		for _, tag := range subnet.Tags {
+			stringKey := awssdk.StringValue(tag.Key)
+			switch stringKey {
+			case tags.RedHatManaged:
+				stringValue := awssdk.StringValue(tag.Value)
+				boolValue, err := strconv.ParseBool(stringValue)
+				if err != nil {
+					reporter.Debugf("An error occurred whilst parsing bool value of '%s' tag, value '%s'", stringKey, stringValue)
+					boolValue = false
+				}
+				hasRHManaged = boolValue
+			}
+			if hasRHManaged {
+				break
```

Comment:
Maybe we can create a slice with all the subnets that were excluded? 
then we can inform the user:
`I: "The following subnets were excluded because they belong to a VPC that is managed by Red Hat: <subnet_ids>"`
So the user knows which subnets were excluded and why.

Reply:
I see, that can be useful. But should we even show it? RH vpcs aren't suppose to be selectable anyway :thinking:

Reply:
It is similar to this thread https://redhat-internal.slack.com/archives/C054PJX8LE8/p1693487192971549
https://issues.redhat.com/browse/OCM-3532.

If the user is expecting to find all subnets displayed, and we are omitting some of them, that can be confusing for the user, they might think it's just a bug on the CLI not displaying the subnets they are looking for.

Reply:
If they provide the subnets using the flags, they are going to get an error: "the subnet doesn't exist", without further explanation.